src/main/java/com/labs/introtoprogramming/lab4/image/formats/bmp/BMPParser.java

```diff
@@ -0,0 +1,56 @@
+package com.labs.introtoprogramming.lab4.image.formats.bmp;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+
+public class BMPParser {
+
+  public static int parseHeader(byte[] headerInfo) throws Exception {
+    if (headerInfo.length != 14) {
+      throw new Exception("File format is not a BMP");
+    }
+    if (headerInfo[0] != 66 || headerInfo[1] != 77) {
+      throw new Exception("File format is not a BMP");
+    }
+
+    int size = sumUpBytes(headerInfo, 2, 6);
+    int offset = sumUpBytes(headerInfo, 10, 14);
+    if (offset < 54 || offset > size) {
+      throw new Exception("File format is not a BMP");
```

Comment:
Is there any way to make exception messages more descriptive? Like "unexpected (invalud) value for offset"